src/Cli/Configuration/DomainConfiguration.php

```diff
@@ -48,7 +48,23 @@ public function getConfigTreeBuilder()
                         ->ifTrue(function ($item) {
                             return !filter_var($item, FILTER_VALIDATE_EMAIL);
                         })
-                        ->thenInvalid('The email "%s" is not valid.')
+                        ->thenInvalid('The email %s is not valid.')
```

Comment:
I would keep the quotes, it avoids spacing problems that are difficult to read and debug.

Reply:
The value injected in the placeholder already contain quote. (see https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Config/Definition/Builder/ExprBuilder.php#L189) 

This pr replace
```diff
- The email ""foo"" is not valid.
+ The email "foo" is not valid.
```